std_msgs/Header header

string name_sensor                  # sensor name for testing
float32 dist_testpiece              # distance from the sensor to the test piece

